candi/cloud-providers/gcp/docs/ENVIRONMENT_RU.md

```diff
@@ -2,24 +2,22 @@
 title: "Cloud provider — GCP: подготовка окружения"
 ---
 
-Для того, чтобы Deckhouse смог управлять ресурсами в Google Cloud, в последнем необходимо создать сервисный аккаунт. Подробная инструкция по этому действию доступна [в документации провайдера](https://cloud.google.com/iam/docs/service-accounts).
+Для того чтобы Deckhouse мог управлять ресурсами, в Google Cloud необходимо создать service account. Далее представлена краткая последовательность действий по созданию service account. Если вам необходима более подробная инструкция, вы можете найти ее [в документации провайдера](https://cloud.google.com/iam/docs/service-accounts).
 
 > **Внимание!** Созданный `service account key` невозможно восстановить, только удалить и создать новый.
 
-Далее представлена краткая последовательность необходимых действий.
+## Настройка через Google Cloud Console
 
-## Настройка через Google cloud console
+Перейдите по [ссылке](https://console.cloud.google.com/iam-admin/serviceaccounts), выберите проект и создайте новый service account (также можно выбрать уже существующий).
 
-Перейдите по [ссылке](https://console.cloud.google.com/iam-admin/serviceaccounts), выберите проект и создайте новый сервисный аккаунт (также можно выбрать уже существующий).
```

Comment:
```suggestion
Перейдите [по ссылке](https://console.cloud.google.com/iam-admin/serviceaccounts), выберите проект и создайте новый сервисный аккаунт (также можно выбрать уже существующий).
```
I think it's more correct. The preposition should also be included in the link.

Reply:
fixed